content/governing-content/understand-content-lifecycle/content-production-template/main-content.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: templates/table
+table:
+  -
+    - |
+        Main content
+  -
+    - |
+    The first sentence sets an expectation for what’s on the page
```

Comment:
to make it clearer for the content authors I would indent this to be consistent

Reply:
done please review